.storybook/themes/news.scss

```diff
@@ -1992,6 +1992,98 @@
 				right-rail-advanced-main-right-rail-bottom: (
 					grid-area: main-right-rail-bottom,
 				),
+				search-results-list: (
+					components: (
+						stack: (
+							gap: var(--global-spacing-5),
+						),
+					),
+				),
+				search-results-list-field: (
+					components: (
+						stack: (
+							gap: var(--global-spacing-5),
+						),
+					),
+				),
+				search-results-list-field-wrapper: (
+					display: grid,
+					gap: var(--global-spacing-4),
+					grid-template: "icon input search" 1fr / var(--global-spacing-7) 1fr auto,
+					components: (
+						button: (
+							grid-area: search,
+						),
+						icon: (
+							align-self: center,
+							grid-area: icon,
+							height: var(--global-font-size-9),
+							justify-self: center,
+							width: var(--global-font-size-9),
```

Comment:
Should be using a spacing value for width not a font size

Reply:
Fixed